### DISCUSSION TITLE
Two value toggle checkbox

### DISCUSSION THREAD

**Discussion by einarpersson** [epwalsh/obsidian.nvim#686] [Q&A]
Hi!

When toggling checkbox it seems to cycle through a range of different checkbox variants.

I just want `[ ]` or `[x]`, can I reconfigure something to get it to toggle between two values instead of 5 or whatever it is currently.

If not, can I change the order, so that it first goes to checked state?

**Comment by joost8** [comment]
I was also wondering about how to configure this.
